libs/ui-lib/lib/ocm/services/ClusterDetailsService.ts

```diff
@@ -70,6 +73,12 @@ const ClusterDetailsService = {
       params.pullSecret = values.pullSecret;
     }
 
+    if (values.platform) {
```

Comment:
Since you are not using the `values.platform` in `values`, it would be better to base the logic only on  `patform`
```suggestion
    if (platform) {
```